### DISCUSSION TITLE
[remark-lint] Automatically fix Markdown

### DISCUSSION THREAD

**Discussion by Denice00** [remarkjs/remark#730] [Q&A] [answered]
Can remark be used to automatically fix the issues in the markdown files that it scans. (similar to prettier --fix)
If yes can you direct me to the docs or give me an alternative to automatically fix markdown readmes based off of the remark markdown rules.

New to coding, was only able to get remark to scan for those issues
- looked at remark-stringify but got no idea how to get it to automatically fix the file.
Current set up 
- using .remarkrc file
```
{
  "settings": {
    "unordered-list": "*",
    "emphasis": "*",
    "strong": "*"
  },
  "plugins": [
    "preset-lint-markdown-style-guide"
  ]
}
```
- running commands from cli so the following is configured on package.json
`"lint-md": "remark . --frail && node index.js",`

**Comment by ChristianMurphy** [comment, marked as answer]
Fixable remark lint rules haven't been implemented yet https://github.com/remarkjs/remark-lint/issues/82

**Reply by Denice00** [reply]
I managed to get it to fix bullet points from - to *, is that the only fix remark has?

**Reply by ChristianMurphy** [reply]
The formatting options are documented here https://github.com/syntax-tree/mdast-util-to-markdown#formatting-options

**Reply by Denice00** [reply]
awesome, thank you!

**Reply by Denice00** [reply]
say is there a way to use prettier with remark --> can prettier fix the markdown files to match the remark formatting standards?

Cause I searched up using remark and prettier but can't figure it out. https://www.npmjs.com/package/remark-preset-prettier

**Reply by ChristianMurphy** [reply]
Prettier will fix to match your prettier settings.
The reset package disables remark lint rules that would flag/warn about prettier formatting.

**Reply by grantaveryatgfs** [reply]
> The formatting options are documented here https://github.com/syntax-tree/mdast-util-to-markdown#formatting-options

@ChristianMurphy that section no longer exists in that README, would this be the correct one now? https://github.com/syntax-tree/mdast-util-to-markdown#options

**Reply by ChristianMurphy** [reply]
Yes, the section is now named options.
